e2e/LongHaul/service/IotHub.cs

```diff
@@ -53,7 +53,7 @@ public async Task RunAsync(CancellationToken ct)
             while (!ct.IsCancellationRequested)
             {
                 await InvokeDirectMethodAsync(++methodCallsCount, ct).ConfigureAwait(false);
-                await SetDesiredPropertiesAsync("methodCallsCount", methodCallsCount, ct).ConfigureAwait(false);
+                await SetDesiredPropertiesAsync("methodCallsCount", methodCallsCount.ToString(), ct).ConfigureAwait(false);
```

Comment:
I am still calling both methods from `RunAsync()` here as we need to pass in the `methodCallsCount` into both methods and increase its value per each loop.

Reply:
`methodCallsCount.ToString()` as we assume all properties are string only.